### ISSUE TITLE
TUFLOW Utilities  - ASC_To_ASC Flags

### ISSUE THREAD

**Issue by itsgupta** [TUFLOW-Support/QGIS-TUFLOW-Plugin#28]
The TUFLOW plugin for QGIS has the ability to link TUFLOW utilities.
When performing differences between two grids, there is an option to specify flags in order to obtain the desired output.

No where on the plugin or the wiki does it mention the "-change" flag. This flag can be very useful when performing certain differences and is already built into asc_to_asc.

I request that this flag is supported with documentation and examples in the QGIS plugin and the TUFLOW Wiki:
[ASC To ASC Wiki](https://wiki.tuflow.com/index.php?title=ASC_to_ASC)

If there are any additional flags that are missing from the documentation, I would like to request that they are are mentioned too in the wiki. It is a good utility and works well. 

Thank you.

**Comment by TUFLOW-Support** [comment]
Yes, I agree that this should be added to the TUFLOW Wiki. I've raised a ticket for this to be added.

**Comment by TUFLOW-Support** [comment]
The -change flag is now documented on the TUFLOW Wiki: https://wiki.tuflow.com/index.php?title=ASC_to_ASC#Difference
